openmc/plotter.py

```diff
@@ -540,3 +602,279 @@ def _calculate_xs_elem_mat(this, types, temperature=294., cross_sections=None,
                                   xs[name][line](energy_grid))
 
     return energy_grid, data
+
+
+def calculate_mgxs(this, types, orders=None, temperature=294.,
+                   cross_sections=None, ce_cross_sections=None,
+                   enrichment=None):
+    """Calculates continuous-energy cross sections of a requested type
+
+    If the data for the nuclide or macroscopic object in the library is
+    represented as angle-dependent data then this method will return the
+    average cross section over all angles.
```

Comment:
geometric average, no flux information is available here to do flux-weighting.  Comment adjusted, however.